### DISCUSSION TITLE
Possible to prevent variable substitution in preheader?

### DISCUSSION THREAD

**Discussion by Saeven** [maizzle/framework#809] [Q&A]
Hello!

I have a template with this header:

```
---
title: "Verify Your Email"
preheader: "Verify your @{{ company_name }} account."
---
```

I would need for this to stay as such in the pre-header that is compiled, e.g.:
```
Verify your {{ company_name }} account.
```

Of course, it gets interpolated and it becomes "Verify your undefined account".

Is this possible?

Thank you!

**Comment by cossssmin** [comment]
Hi there, yes, double-ignore it: `@@{{ company_name }}`

Need to mention this in the [docs](https://maizzle.com/docs/templates#ignoring-expressions), what happens is that it gets evaluated both in the template and in the layout because of the way the templating engine works.
